modules/admin_manual/pages/installation/manual_installation/index.adoc

```diff
@@ -1,13 +1,13 @@
 = Manual Installation
 
-In this section, you will find *Installation Guides* you need for installing ownCloud manually.
+In this section, you find *Installation Guides* for manually setting up ownCloud.
 
-Please consider the following when deciding which path of installation you are looking at:
+Consider the following before deciding on a path of installation:
 
-The *Detailed Installation Guide* gives more detailed information and describes individual setup
+The xref:installation/manual_installation/manual_installation.adoc[Detailed Installation Guide] gives more detailed information and describes individual setup
 possibilities. It may not fit all audiences as it requires a deeper background knowledge. As a
 bonus, it provides ready to use scripts which need some preconfiguration to run successfully.
-The scripts are "as it is" and no responsibility is taken.
+The scripts are "as it is" and we cannot take any responsibility for them working properly.
 
-The *Quick Installation Guides* are _very_ helpful when doing standard setups and all commands
-necessary are setup for copy/paste. Using these guides, you are as it says, quickly up and running.
+The xref:installation/quick_guides/index.adoc[Quick Installation Guides] are useful for standard setups. All commands
+necessary are provided for you to simply copy&paste, but little information is provided beyond bare instructions. If you use these guides, ownCloud will be up and running in very little time.
```

Comment:
I would not write about a _standard_ but a _basic_ setup - which is true. Standard is dependent what you want and differs personally. In addition, I suggest to add a text that when using _basic_,  this may not fit using all possibilities owncloud offers.